rpcd/playbooks/roles/rpc_maas/tasks/rabbitmq_user.yml

```diff
@@ -51,6 +46,6 @@
     write_priv: ".*"
     tags: "administrator"
     state: "present"
-  with_items: vhost_json
+  with_items: vhost_list.json
```

Comment:
Where does this file come from? ``vhost_list`` is the result register in the above task so I'm not sure why this is changing.